lib/client.py

```diff
@@ -136,7 +137,9 @@ def backend_push_authenticate(self, user_id):
         return True
 
     def provision_user(self, phone_number, email_address, display_name):
-        logger.debug('Provisioning user with the following details: phone number: {}, email address: {}, display name: {} '.format(
+        logger.debug(('Provisioning user with the following details: phone number: {},'
+                      'email address: {},'
+                      'display name: {} ').format(
```

Comment:
the trailing space is probably a typo, also if we're going to do this, let's move phone number on its own line as well

Reply:
Done